### PR TITLE
8 modification du fichier css

### PR DESCRIPTION
**Pull request by LaserHuy**

### CODE REVIEW

**Review by yazidsef — APPROVED** [review]

File: public/index.php [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?php
 
+$name = 'Oxygen';
```
Comment (Benja-san) [inline public/index.php:3]: this variable has no place in this file

**Review by Benja-san — CHANGES_REQUESTED** [review]
Why didi you set all those classNames into your global css file ?

File: public/index.php (hunk 1 above)
Comment (LaserHuy) [reply public/index.php:3]: Where can I put this variable? In the HomeController or layout.html.twig ?

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -6,10 +6,19 @@
...
-    <title>{% block title %}{% endblock %}</title>
+    <title>{% block title %}{{ name }}{% endblock %}</title>
 
     <link rel="stylesheet" href="/assets/css/style.css">
-    {% block stylesheet %}{% endblock %}
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Playpen+Sans:wght@500;900&amp;display=swap" rel="stylesheet">
+    
+    {% block stylesheet %}
+        <style>
```
[the remark this answers is not in the document]
Comment (LaserHuy) [reply src/View/layout.html.twig:16]: ok I got it

File: src/View/layout.html.twig [hunk 3]
```diff
@@ -6,10 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" href="/assets/images/favicon.png">
 
-    <title>{% block title %}{% endblock %}</title>
+    <title>{% block title %}{{ name }}{% endblock %}</title>
 
     <link rel="stylesheet" href="/assets/css/style.css">
-    {% block stylesheet %}{% endblock %}
+    <script src="https://cdn.tailwindcss.com"></script>
```
[the remark this answers is not in the document]
Comment (LaserHuy) [reply src/View/layout.html.twig:12]: I try to install Tailwind Css for CDN in the website it is said that is to insert this script to the head tag

**Pull request merged**